src/test/resources/com/fulcrumgenomics/vcf/intervalList.1-10.assess_phasing_metrics.block_lengths.txt

```diff
@@ -0,0 +1,6 @@
+is_truth	length	count
```

Comment:
Just thinking aloud - would it make sense to have the first column be `dataset` or something like that with values being `truth` and `called`?  I think that might be easier to consume, and also leaves room if you wanted to add more in future (e.g. the intersection of the truth and called phase blocks).